pkg/consts/views.go

```diff
@@ -55,6 +55,19 @@ function(doc) {
 }`,
 }
 
+// FilesByParentView is the view used for fetching files referenced by a
+// given document
+// TODO change me for flag hidden
+var FilesByParentView = &couchdb.View{
+	Name:    "by-parent-type-name",
```

Comment:
Won't it be more practical to use the new view for iteraring? If yes, we can keep this for later (just add a TODO, so that we don't forget)